### DISCUSSION TITLE
Is there a winget equivalent of "apt --download-only"

### DISCUSSION THREAD

**Discussion by sjsawatzky** [microsoft/winget-cli#2730] [Q&A]
Many of my users have poor internet, and I'm wondering if there is a way to automatically download/cache the files so they can be installed later all at once. Currently, I can run "winget update --all" but it might take hours, doing one download and then one install, the next download and install, etc.   

I'd like to be able to cache download/cache the installers so I can run them all at once.

**Comment by Trenly** [comment]
At the time, no. This is being tracked by https://github.com/microsoft/winget-cli/issues/658
